From 74c5bdae732484d6e41a24ec89c6ba35d5882e84 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:04:53 -0400
Subject: [usb] xhci: Remove unused variable 'addr' in inc_deq() and inc_enq()

Message-id: <1410887463-20674-222-git-send-email-dzickus@redhat.com>
Patchwork-id: 95416
O-Subject: [RHEL7 PATCH 221/591] xhci: Remove unused variable 'addr' in inc_deq() and inc_enq().
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 599459d8230bd6af9c354e00ad6608c43b6f7426
Author: Lin Wang <bupt.wanglin@gmail.com>
Date:   Tue Dec 17 17:59:54 2013 +0000

    xhci: Remove unused variable 'addr' in inc_deq() and inc_enq().

    This patch remove unused variable 'addr' in inc_deq() and inc_enq().

    Signed-off-by: Lin Wang <lin.x.wang@intel.com>
    Signed-off-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/xhci-ring.c b/drivers/usb/host/xhci-ring.c
index c06df38..09b2b55 100644
--- a/drivers/usb/host/xhci-ring.c
+++ b/drivers/usb/host/xhci-ring.c
@@ -156,8 +156,6 @@ static void next_trb(struct xhci_hcd *xhci,
  */
 static void inc_deq(struct xhci_hcd *xhci, struct xhci_ring *ring)
 {
- unsigned long long addr;
-
  ring->deq_updates++;
 
  /*
@@ -186,8 +184,6 @@ static void inc_deq(struct xhci_hcd *xhci, struct xhci_ring *ring)
    ring->dequeue++;
   }
  } while (last_trb(xhci, ring, ring->deq_seg, ring->dequeue));
-
- addr = (unsigned long long) xhci_trb_virt_to_dma(ring->deq_seg, ring->dequeue);
 }
 
 /*
@@ -212,7 +208,6 @@ static void inc_enq(struct xhci_hcd *xhci, struct xhci_ring *ring,
 {
  u32 chain;
  union xhci_trb *next;
- unsigned long long addr;
 
  chain = le32_to_cpu(ring->enqueue->generic.field[3]) & TRB_CHAIN;
  /* If this is not event ring, there is one less usable TRB */
@@ -264,7 +259,6 @@ static void inc_enq(struct xhci_hcd *xhci, struct xhci_ring *ring,
   ring->enqueue = ring->enq_seg->trbs;
   next = ring->enqueue;
  }
- addr = (unsigned long long) xhci_trb_virt_to_dma(ring->enq_seg, ring->enqueue);
 }
 
 /*
-- 
1.7.1